src/main.rs

```diff
@@ -67,7 +73,17 @@ fn validate_simulation() {
 
 /// Simulate the rabbit population
 fn simulate(starting_rabbits: i128) -> i32 {
-    todo!()
+    let mut remainingRabits: i128 = starting_rabbits;
+    let mut numDays: i32 = 0;
+    while remainingRabits > 1 {
+        numDays += 1;
+        if remainingRabits % 2 != 0 {
+            remainingRabits = 3 * remainingRabits + 1;
+        } else if remainingRabits % 2 == 0 {
+            remainingRabits = remainingRabits / 2;
```

Comment:
I didn't think of it for my solution, but I did see `remainingRabits /= 2;` in some other students' code, and that was pretty neat :)